src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -317,19 +355,31 @@ private WildcardQuery CalculateFieldWeight(string fieldName, string searchTerm,
         {
             WildcardQuery query;
 
+            var termText = fieldName == nameof(LuceneConfig.NodeFieldsEnum.NameSplitted) ? searchTerm + "*" : "*" + searchTerm + "*";
```

Comment:
Would you add some comments on this line?

Reply:
I've added a comment in the next commit: https://github.com/DynamoDS/Dynamo/pull/14663/commits/a3f7bd60ce8964c12caa157c803fa0e7ae800937